pages/gear.php

```diff
@@ -16,9 +16,9 @@ function gear(): void {
 		$max_equipped_armor++;
 		$max_equipped_weapons++;
 	}
-	if($player->forbidden_seal && $player->forbidden_seal['level'] >= 2) {
-		$max_equipped_armor++;
-		$max_equipped_weapons++;
+	if($player->forbidden_seal) {
+		$max_equipped_armor += System::$premium_benefits[$player->forbidden_seal['level']]['armor_equips'];
```

Comment:
```suggestion
		$max_equipped_armor += System::$premium_benefits[$player->forbidden_seal['level']]['extra_armor_equips'];
```
I think this clarity may be more clear when we're just editing an array in the System class. Similar comment for `'weapon_equips'` and `'jutsu_equips'`